deploy/scripts/edge-setup.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
```

Comment:
Need to undo this, the script should fail at any point  needed.   Or we should add individual return code checks.